### ISSUE TITLE
Accessibility Feedback for Staging Review | Anchor Tag Pilot on CoronaFAQ Page | C+L Team

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#23384]
## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

**Comment by 1Copenut** [comment]
VSP concurs with @joshkimux feedback!

**Comment by SKasimow** [comment]
Hi @joshkimux - I am not seeing your feedback; can you let me know if it should be visible on this ticket?

**Comment by joshkimux** [comment]
@SKasimow , if you have zenhub installed as an extension on chrome, they should all appear!
I've listed them as links below as well:
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/23919
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/23963
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/23968
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/23976

I should have flagged this to you earlier as we completed this last week! Some of them (hopefully) may have been resolved by now :)

**Comment by SKasimow** [comment]
@joshkimux I'm using Safari and didn't see yesterday....but today I am able to see them! Thx

**Comment by joshkimux** [comment]
@SKasimow and @youngfreezyVA 

I'll follow back up on this first thing next week to see if we can find a fix.

Not a launchblocker, but I caught what's probably a `defect-level-2` in this ticket. The feedback "link copied" is not immediately provided to the user upon interacting with the button and can get easily lost when announced by a screen reader.

My first thought was that it could be fixed by setting `aria-live` to `assertive`, but it didn't help too much. I've provided demos below showcasing the behavior.

## Current, with polite

https://user-images.githubusercontent.com/14154792/118320536-64a72080-b4ca-11eb-8c09-12b57e17696f.mov

On activation, `polite` will announce the button name again "Copy a link for Can I get a COVID-19 vaccine at VA?, button" followed by "3" (I think the heading level?) "Can I get a COVID-19 vaccine at VA? Copy a link for Can I get a COVID-19 vaccine at VA? **Link copied**" 

## Test, with assertive

https://user-images.githubusercontent.com/14154792/118320529-61ac3000-b4ca-11eb-9ca9-616455dabbbd.mov

On activation, `assertive` won't repeat the button name again, but will announce  "3" (I think the heading level?) "Can I get a COVID-19 vaccine at VA? Copy a link for Can I get a COVID-19 vaccine at VA? **Link copied**"

**Comment by SKasimow** [comment]
Hi @joshkimux - can you close the accessibility issues associated with this effort? We have removed the anchor links in the accordion and will be removing the one in the open question during our next sprint. Thx!

**Comment by joshkimux** [comment]
Will do!
